src/ShallowTraversal.js

```diff
@@ -29,7 +29,7 @@ export function childrenOfNode(node) {
 
 export function hasClassName(node, className) {
   const classes = propsOfNode(node).className || '';
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  return ` ${classes.trim()} `.indexOf(` ${className} `) > -1;
```

Comment:
trim won't catch `foo\nbar`, will it?

I think you may instead want to `.replace(/\s*/g, ' ')`